libraries/SuplaDevice/supla-common/srpc.h

```diff
@@ -22,27 +22,33 @@
 #include <stdio.h>
 #include "eh.h"
 #include "proto.h"
+#if defined(ESP32)
+#include <esp8266-compat.h>
+#endif
 
 #ifdef __ANDROID__
 #define SRPC_EXCLUDE_DEVICE
 #endif /*__ANDROID__*/
 
-#ifdef ESP8266
+#if defined(ESP8266) || defined(ESP32)
 
-#ifdef ARDUINO_ARCH_ESP8266
+#if defined(ARDUINO_ARCH_ESP8266) || defined(ARDUINO_ARCH_ESP32)
```

Comment:
....
Good as you will update supla-common from supla-core / supla-common again

Reply:
done